src/app/router/Prompt/Prompt.jsx

```diff
@@ -69,11 +65,19 @@ class Prompt extends Component {
                   />
                 }
               />
+              <Route
+                path="invoice"
+                element={
+                  <MakeInvoice
+                    invoice={this.state.args.invoice}
```

Comment:
maybe we should call this invoiceAttributes or something like that to make clear what is in there? (in this case no bolt11 payment request/invoice)